### DISCUSSION TITLE
SBOM generation guidance

### DISCUSSION THREAD

**Discussion by Ind0mitable** [aquasecurity/trivy-operator#1413] [Help & Support]
Hello! 

 I have the latest version on the trivy-operator deployed in my openshift cluster, however I am still having a hard time generating an SBOM report. I found the CRD that handles the generation of the report; I am having a hard time actually generating the SBOM report itself. I attempted to generate the report using one of the scans, as well as trying to template together a YAML for it to ingest. Still no success.  Is there any guidance or documentation on how to generate an SBOM report? This is a key feature my team is looking for, and I am relatively new to integrating solutions into k8s/openshift. So thank you for your patience.

Thanks!

**Comment by chen-keinan** [comment]
@Ind0mitable if you run `kubectl get sbom --all-namespaces` you are getting the reports ? 
which trivy-operator version are you using ?

**Reply by Ind0mitable** [reply]
I am using the latest version v0.15.1. When I run kubectl get sbom --all-namespaces it returns "No resources found".

**Reply by chen-keinan** [reply]
@Ind0mitable can you please confirm the [sbom flag](https://github.com/aquasecurity/trivy-operator/blob/main/deploy/helm/templates/deployment.yaml#L72) is enabled on deployment:

**Reply by Ind0mitable** [reply]
I can confirm the sbom flag is enabled. I followed the release instructions for the latest version, and deleted the crd before upgrading. I have since redeployed a fresh v0.15.1, with no success.

**Reply by chen-keinan** [reply]
can you  please share your configmap?

**Reply by chen-keinan** [reply]
Closing the discussion and opened an issue #1417, please update your input in the issue
